.github/workflows/onPushToMain.yml

```diff
@@ -7,12 +7,50 @@ on:
 
 jobs:
   release:
-    uses: oclif/github-workflows/.github/workflows/githubRelease.yml@main
-    secrets: inherit
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+      - name: Check if version already exists
+        id: version-check
+        run: |
+          package_version=$(node -p "require('./package.json').version")
+          exists=$(gh api repos/${{ github.repository }}/releases/tags/v$package_version >/dev/null 2>&1 && echo "true" || echo "")
```

Comment:
This will not work as expected if people are not tagging with a `v` in the tag name.

<img width="1352" alt="Screenshot 2024-04-22 at 11 53 50 AM" src="https://github.com/oclif/hello-world-esm/assets/1715111/3fdf8bc3-1a99-404e-8f21-045ede881cee">

Reply:
I'm not particularly worried about this since I'm not trying to create a workflow that will work for all people in all circumstances. It's just supposed to be a happy path to get people started. If people want `v`-less tags, then they can update all the actions accordingly